src/components/cuda/cupti_dispatch.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file    cupti_dispatch.h
+ * @author  Anustuv Pal
+ *          anustuv@icl.utk.edu
+ */
+
+#ifndef __CUPTI_DISPATCH_H__
+#define __CUPTI_DISPATCH_H__
+
+#include "cupti_utils.h"
```

Comment:
I think we should have this wrapped with `cuptid_` equivalents. This cupti_utils interface should be internal to the cupti wrapper layer.